### ISSUE TITLE
Request: Build instructions in README

### ISSUE THREAD

**Issue by aayux** [skyman/particles#1]
A short readme with dependencies and build requirements would be really helpful! I'm fairly new to Java and would love to get this project running on my machine.

Thanks!

**Comment by cgokmen** [comment]
Apologies for missing this. Have you been able to get it to work?

**Comment by cgokmen** [comment]
If you download IntelliJ IDEA and import the directory as a project, you should be able to easily run it. There's a couple separate runners - there's the main, GUI app and there's some others that run experiments with sets of values. Search for "public static void main" in the project to find all of these.

Also, may I ask - what are you using this for?

**Comment by aayux** [comment]
Hey, thanks for your response! Following your advice, I tried running with IntelliJ IDEA as well, but the same issue persists. 

My understanding is that this has to do with JavaFX and my operating system version [see [here](https://stackoverflow.com/questions/56166267/how-do-i-get-java-fx-running-with-openjdk-8-on-ubuntu-18-04-2-lts/56166582#56166582) and [here](https://stackoverflow.com/a/15283999)]. Following the link, I was _almost_ able to install the JavaFX plugin for Maven, but that build eventually failed too -- this time because of a JDK mismatch.

Anyhow, at the end I figured, I could just read the code and figure most of it out from syntax. It's also why I closed the issue.

>Also, may I ask - what are you using this for?

I'm starting work on SOPS and wanted a point of reference (other implementations and such), and yours was (one of?) the only open source projects.
